Plugins/GeoModel/include/Acts/Plugins/GeoModel/GeoModelTree.hpp

```diff
@@ -18,7 +18,7 @@ namespace Acts {
 
 struct GeoModelTree {
   std::shared_ptr<GeoModelIO::ReadGeoModel> geoReader = nullptr;
-  GeoVPhysVol* worldVolume = nullptr;
+  const GeoVPhysVol* worldVolume = nullptr;
```

Comment:
Will lead to clashes with developements by @dimitra97

Reply:
I swore I fixed this before, Will make it consistent with her